### PR TITLE
Prepare contrib docs for PTL

### PR DESCRIPTION
**Pull request by kamil-kaczmarek**
* Dropped all contrib docs
* Updated PTL legacy logger docstrings
* contrib-docs is now one-pager.

### CODE REVIEW

**Comment by kamil-kaczmarek** [comment]
build the docs locally for better review experience:

* git clone and checkout `kk/ptl` branch.
* install `pip install -r docs_requirements.txt`
* `cd docs`
* `make clean & make html`

@Herudaio

File: neptunecontrib/monitoring/pytorch_lightning.py [hunk 1]
```diff
@@ -29,27 +29,28 @@
 
 class NeptuneLogger(LightningLoggerBase):
     r"""
-    Log using `Neptune <https://neptune.ai>`_.
+    .. warning::
+        This is LegacyLogger implementation that is under maintenance support only.
```
Comment (Herudaio) [inline neptunecontrib/monitoring/pytorch_lightning.py:33]:
```suggestion
        This is a LegacyLogger implementation that is under maintenance support only.
```

File: neptunecontrib/monitoring/pytorch_lightning.py [hunk 2]
```diff
@@ -29,27 +29,28 @@
...
+        No new updates will be made to this logger.
 
-    Install it with pip:
+        Please visit integration
```
Comment (Herudaio) [inline neptunecontrib/monitoring/pytorch_lightning.py:36]:
```suggestion
        Please visit the current PyTorch Lightning integration
```

File: neptunecontrib/monitoring/pytorch_lightning.py [hunk 3]
```diff
@@ -29,27 +29,28 @@
...
+        `docs <https://docs.neptune.ai/integrations-and-supported-tools/model-training/pytorch-lightning>`_
+        to learn about the latest, fully supported version.
 
-    .. code-block:: bash
-
-        pip install neptune-client
-
-    The Neptune logger can be used in the online mode or offline (silent) mode.
+    The legacy Logger can be used in the online mode or offline (silent) mode.
```
Comment (Herudaio) [inline neptunecontrib/monitoring/pytorch_lightning.py:40]:
```suggestion
    The legacy Logger can be used in online mode or silent mode.
```
I would avoid using "offline" not for the new offline mode unless we cannot avoid it (like param name :D).

**Review by Herudaio — COMMENTED** [review]
Minor comments.

File: neptunecontrib/monitoring/pytorch_lightning.py [hunk 4]
```diff
@@ -29,27 +29,28 @@
...
+        Please visit the current PyTorch Lightning integration
+        `docs <https://docs.neptune.ai/integrations-and-supported-tools/model-training/pytorch-lightning>`_
+        to learn about the latest, fully supported version.
 
-    .. code-block:: bash
-
-        pip install neptune-client
-
-    The Neptune logger can be used in the online mode or offline (silent) mode.
+    The legacy Logger can be used in online mode or silent mode.
```
Comment (kamil-kaczmarek) [inline neptunecontrib/monitoring/pytorch_lightning.py:40]:
```suggestion
    The legacy Logger can be used in online mode or offline (silent) mode.
```

**Review by Herudaio — APPROVED** [review]

**Pull request merged**